### DISCUSSION TITLE
The Sound for Local Notification does not support in Android 26+, Can Anyone tell me a work around for working that?

### DISCUSSION THREAD

**Discussion by namrata-Zimbaroos** [ionic-team/capacitor#4452] [Q&A]
I am using the Local Notification Plugin of Capacitor, I want to add a sound there But the "sound" Local Notification "schedule" works till Android 26(Android O). I need a workaround or any possibility to make it work for all the Android Versions which are available. I have tried to create a channel and passing that channel Id to schedule but it does not work. Could you please help me with that?

**Comment by jcesarmobile** [comment]
On Android 26 you have to set the sound of the channel, you can set the sound of the default channel or use a custom channel.
You can't change the channel sound after creation, so if you already installed the app, you'll have to uninstall it and install it again with the sound configured.

https://capacitorjs.com/docs/apis/local-notifications#local-notifications-configuration-android-only
